go/test/endtoend/multipooler_manager_test.go

```diff
@@ -2098,3 +2168,574 @@ func TestConfigureSynchronousReplication(t *testing.T) {
+		assert.Len(t, status.SyncReplicationConfig.StandbyIds, 2)
+		assert.True(t, containsStandbyIDInConfig(status.SyncReplicationConfig, "test-cell", "standby1"))
+		assert.True(t, containsStandbyIDInConfig(status.SyncReplicationConfig, "test-cell", "standby2"))
+
+		t.Log("ADD operation verified successfully")
+
+		// Cleanup
+		resetReq := &multipoolermanagerdata.ConfigureSynchronousReplicationRequest{
+			SynchronousCommit: multipoolermanagerdata.SynchronousCommitLevel_SYNCHRONOUS_COMMIT_ON,
+			SynchronousMethod: multipoolermanagerdata.SynchronousMethod_SYNCHRONOUS_METHOD_FIRST,
+			NumSync:           0,
+			StandbyIds:        []*clustermetadatapb.ID{},
+			ReloadConfig:      true,
+		}
+		_, err = primaryManagerClient.ConfigureSynchronousReplication(utils.WithShortDeadline(t), resetReq)
+		require.NoError(t, err)
+	})
+
+	t.Run("UpdateSynchronousStandbyList_Add_Idempotent", func(t *testing.T) {
+		t.Log("Testing UpdateSynchronousStandbyList ADD operation is idempotent...")
+
+		// Configure with two standbys
+		configReq := &multipoolermanagerdata.ConfigureSynchronousReplicationRequest{
+			SynchronousCommit: multipoolermanagerdata.SynchronousCommitLevel_SYNCHRONOUS_COMMIT_ON,
+			SynchronousMethod: multipoolermanagerdata.SynchronousMethod_SYNCHRONOUS_METHOD_FIRST,
+			NumSync:           1,
+			StandbyIds: []*clustermetadatapb.ID{
+				makeMultipoolerID("test-cell", "standby1"),
+				makeMultipoolerID("test-cell", "standby2"),
+			},
+			ReloadConfig: true,
+		}
+		_, err := primaryManagerClient.ConfigureSynchronousReplication(utils.WithShortDeadline(t), configReq)
+		require.NoError(t, err)
+
+		// Wait for config to converge
+		waitForSyncConfigConvergenceWithClient(t, primaryManagerClient, func(config *multipoolermanagerdata.SynchronousReplicationConfiguration) bool {
+			return config != nil && len(config.StandbyIds) == 2 &&
+				containsStandbyIDInConfig(config, "test-cell", "standby1") &&
+				containsStandbyIDInConfig(config, "test-cell", "standby2")
+		}, "Initial config should converge")
+
+		initialStatus := getPrimaryStatusFromClient(t, primaryManagerClient)
+		t.Log("Initial configuration verified")
+
+		// Try to ADD a standby that already exists
+		updateReq := &multipoolermanagerdata.UpdateSynchronousStandbyListRequest{
+			Operation:     multipoolermanagerdata.StandbyUpdateOperation_STANDBY_UPDATE_OPERATION_ADD,
+			StandbyIds:    []*clustermetadatapb.ID{makeMultipoolerID("test-cell", "standby1")},
+			ReloadConfig:  true,
+			ConsensusTerm: 0,
+			Force:         true,
+		}
+		_, err = primaryManagerClient.UpdateSynchronousStandbyList(utils.WithShortDeadline(t), updateReq)
+		require.NoError(t, err, "ADD should be idempotent")
+
+		// Wait for config to settle - should remain unchanged (idempotent)
+		// We wait for the same configuration to ensure pg_reload_conf() completes
+		waitForSyncConfigConvergenceWithClient(t, primaryManagerClient, func(config *multipoolermanagerdata.SynchronousReplicationConfiguration) bool {
```

Comment:
Since there's no change in the value, does this wait do anything?

Would it also make sense somewhere in this test case to try to add both an existing a new a new standby in a single request?